src/transformStateWithClones.js

```diff
@@ -7,7 +7,30 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateObj = { ...state };
+  const actionsArr = [];
+
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
```

Comment:
you should take a look at the checklist again

Reply:
done)